src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -21,6 +21,62 @@ extern "C" {
 typedef halide_hexagon_remote_handle_t handle_t;
 typedef halide_hexagon_remote_buffer buffer;
 
+#define MAX_WORKER_THREADS 4
+#define NUM_WORKER_THREADS_TO_CREATE 3
+#define STACK_SIZE 4096
```

Comment:
This should be bigger, Halide allocates allocations up to 16 kb on the stack, and there might be several of those. I think we should set it to 256 KB. If this means it must be heap allocated, that's OK.